example/mongoose-multiple-connections.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express')
+const mongoose = require('mongoose')
+const session = require('express-session')
+const MongoStore = require('connect-mongo').default
+
+// App Init
+const app = express()
+const port = 3000
+
+// Starting Server
+app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`)
+})
+
+// Mongoose Connection
+const appDBConnection = mongoose
+  .createConnection(process.env.APP_DB_URI, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  })
+  .then((connection) => {
+    console.log('Connected to AppDB.')
+    return connection
+  })
+
+const sessionDBConnection = mongoose
+  .createConnection(process.env.SESSIONS_DB_URI, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  })
+  .then((connection) => {
+    console.log('Connected to SessionsDB.')
+    return connection
+  })
+
+// Session Init
+const sessionInit = (clientPromise) => {
+  app.use(
+    session({
+      store: MongoStore.create({
+        clientPromise,
```

Comment:
The new version can now set `promise: promise`. Thus remove the need of `Promise.resolve` in outer scope.